### PR TITLE
Fix broken docs links and redirects

### PR DESCRIPTION
**Pull request by cameel**

### CODE REVIEW

File: docs/contributing.rst [hunk 1]
```diff
@@ -408,7 +408,7 @@ local slang and references, making your language as clear to all readers as poss
 
 * `Simplified technical English <https://en.wikipedia.org/wiki/Simplified_Technical_English>`_
 * `International English <https://en.wikipedia.org/wiki/International_English>`_
-* `British English spelling <https://en.oxforddictionaries.com/spelling/british-and-spelling>`_
+* `British English spelling <https://web.archive.org/web/20220324094038/https://www.lexico.com/grammar/british-and-spelling>`_
```
Comment (cameel) [inline docs/contributing.rst:411]: This page seems no longer available, not just moved. Not much time for selecting a better source just before the release so I decided to link to the last available snapshot on archive.org instead.

**Comment by cameel** [comment]
There are two other broken links:
- https://www.visualstudio.com/downloads/#build-tools-for-visual-studio-2019
    - MS changes this all the time. Not sure it's even still available.
- https://swarm-gateways.net/bzz:/16c5f09109c793db99fe35f037c6092b061bd39260ee7a677c8a97f18c955ab1/
    - Not sure if the gateway is just temporarily down or if we have to find a new one.

There are also a few more redirects, where I'm not sure if we should switch the link or not (e.g. `spdx.org` is now apparently `spdx.dev` for some files we link to but some are still served from the old domain). Probably not something we want to sort out just before the release.

**Comment by r0qs** [comment]
>     * https://swarm-gateways.net/bzz:/16c5f09109c793db99fe35f037c6092b061bd39260ee7a677c8a97f18c955ab1/
>       
>       * Not sure if the gateway is just temporarily down or if we have to find a new one.

They changed their gateway a while ago: https://gateway.ethswarm.org/access

But if we are using the old swarm client it will not be compatible anymore. The new client is called [bee](https://github.com/ethersphere/bee), and it requires a very different setup than the previous swarm versions. Even the hashes algorithms used changed, I think (not sure).

We may need to update solc-bin to use this instead: https://github.com/ethersphere/bee-js/blob/master/src/chunk/bmt.ts#L27

**Review by matheusaaguiar — APPROVED** [review]
Checked that all replaced links are indeed working.

**Review by r0qs — APPROVED** [review]
I'm fine with the changes. Maybe we could remove the broken links though, and create an issue to fix them later.

**Pull request merged**